tasks/collect-collectors-data/README.md

```diff
@@ -0,0 +1,21 @@
+# collect-collectors-data
+
+Tekton task to collect the information related to collectors.
+
+The purpose of this task is to collect all the data and supply it to the other task in the pipeline by creating
+json files for the different resources.
+
+A task result is returned for each resource with the relative path to the stored JSON for it in the workspace. There is
+also a task result for the fbcFragment extracted from the snapshot's first component.
+
+## Parameters
+
+| Name                 | Description                                 | Optional | Default value |
+|----------------------|---------------------------------------------|----------|---------------|
+| previousRelease      | Namespaced name of the previous Release     | No       | -             |
+| release              | Namespaced name of the Release              | No       | -             |
+| releasePlan          | Namespaced name of the ReleasePlan          | No       | -             |
+| releasePlanAdmission | Namespaced name of the ReleasePlanAdmission | No       | -             |
+
+## Changes in 0.1
```

Comment:
I don't think we need this (and if anything, it would be 0.1.0).

Reply:
Agree.